app_kitchen/templates/index.html

```diff
@@ -16,26 +16,33 @@
 <body>
 
   <nav class="navbar">
```

Comment:
Sematic HTML header - add a header tag to hold the nav bar.
Resource: https://internetingishard.com/html-and-css/semantic-html/